lib/exandra.ex

```diff
@@ -327,6 +327,39 @@ defmodule Exandra do
 
     {keyspace, conn}
   end
+
+  @doc """
+  Streams the results of a simple query or a prepared query to the given callback.
```

Comment:
**Nitpick (non-blocking)**: I only bring this up because this is OSS and eventually (hopefully) people will use this: we usually use the first line of docs as a "header", because it's what shows up in the compacted lists of functions in ExDoc.

```suggestion
  Streams the results of a simple query or a prepared query to the given callback.

```